Client/src/Pages/AdvancedSettings/index.jsx

```diff
@@ -72,189 +74,210 @@ const AdvancedSettings = ({ isAdmin }) => {
 		setLocalSettings({ ...localSettings, logLevel: newLogLevel });
 	};
 
-	const handleChange = (event) => {
-		const { value, id } = event.target;
-		setLocalSettings({ ...localSettings, [id]: value });
-	};
+  const handleChange = (event) => {
+    const { value, id } = event.target;
+    setLocalSettings({ ...localSettings, [id]: value });
+    const { error } = advancedSettingsValidation.validate(
+      { [id]: value },
+      {
+        abortEarly: false,
+      }
+    );
+    setErrors((prev) => {
+      return buildErrors(prev, id, error);
+    });    
+  };
```

Comment:
Validating on change can be bad for performance, because we are running validation and state changes even if the user is filling correctly. 
I would suggest we validate on submit.

Reply:
That can be a  design decision made before, I've seen both